Source/Core/VideoCommon/VertexManagerBase.cpp

```diff
@@ -344,6 +344,17 @@ void VertexManagerBase::Flush()
 
   m_is_flushed = true;
 
+  if (xfmem.numTexGen.numTexGens != bpmem.genMode.numtexgens ||
```

Comment:
Does it hurt that this block (which can early return) is _after_ `m_is_flushed = true` ?

Reply:
Yeah, we want to skip the object, i.e. set it as flushed.